xcube_cci/chunkstore.py

```diff
@@ -1059,3 +1046,4 @@ def common_divisors(orig_number: int) -> List[int]:
             divisors.append(int(orig_number / factor))
         factor -= 1
     return sorted(divisors, reverse=True)
+    # return sorted(divisors, reverse=False)
```

Comment:
Should this line be removed before merging into main? or leave a comment, why this is commented :)